.github/workflows/ci.yml

```diff
@@ -42,7 +42,7 @@ jobs:
         env:
           DOCKER_IMAGE_SERVER: ${{ steps.prep.outputs.tagged_image }}
         run: |
-          docker-compose -f ./gh-docker-compose.yml run --rm server bash -c 'wait-for-it db:5432 && ./manage.py graphql_schema --out /ci-share/schema-latest.graphql' &&
+          docker compose -f ./gh-docker-compose.yml run --rm server bash -c 'wait-for-it db:5432 && ./manage.py graphql_schema --out /ci-share/schema-latest.graphql' &&
```

Comment:
Is there any reason we are changing this?

Reply:
As per Saurav: docker-compose doesnot work in github actions so docker compose is used instead of docker-compose @susilnem